openslides/utils/rest_api.py

```diff
@@ -155,14 +157,40 @@ def get_serializer_class(self) -> Type[Serializer]:
         Overridden method to return the serializer class given by the
         access permissions container.
         """
-        if self.get_access_permissions() is not None:
-            serializer_class = self.get_access_permissions().get_serializer_class(self.request.user)  # type: ignore
+        model = self.get_queryset().model  # type: ignore
+        try:
+            return model_serializer_classes[model]
+        except AttributeError:
+            return super().get_serializer_class()  # type: ignore
```

Comment:
Hm. What is the usecase of this line? If there isn't any raise RuntimeError or something like this.

Reply:
I added a comment